content/how-to/account-management/change-account-information/index.md

```diff
@@ -15,30 +15,41 @@ slug: change-account-information
 This article describes the various administrative account changes that you can
 request and the appropriate steps needed to complete them.
 
-### Dedicated customers
+### Dedicated Customers
 
-Dedicated customers can download the **Primary Contact Change** form in the
-[MyRackspace Control Panel](https://login.rackspace.com). For instructions
-about how to find the form, see the **Dedicated customers** section of [Change
-to a new primary contact](/support/how-to/change-to-new-primary-contact).
+You can download the **Primary Contact Change** form by using the following steps: 
 
-If you are the current primary contact for your account, you can update the
-primary contact information by selecting and editing the **Primary Contact
-Information** in your **Account Settings**.
+**Step 1.** Log in to the Rackspace Technology Portal <https://login.rackspace.com/>.
 
-Request the **Service Transfer** form from your account manager.
+**Step 2.** In the top navigation panel, click **Account >> Docs and Forms**.
+
+<img width="359" alt="Docs and Forms option available under the Account navigation" src="accdocs&forms.png">
```

Comment:
Actually images are not rendering even in the preview deployment.
This is what it looks like.
![Screen Shot 2022-10-19 at 11 29 20 AM](https://user-images.githubusercontent.com/14296817/196750330-e478108e-f70f-4011-b09e-32dbca59de9b.png)

So you will have to use the long src path. Please update.

Reply:
Ok Sir

Reply:
Hey Steven, I tried with source path but image was broken. 
When I did preview deployment with the same source, I am able to see the images.
https://deploy-preview-1419--docs-support-how-to.netlify.app/support/how-to/change-account-information/